src/pages/guides/develop/coi.md

```diff
@@ -0,0 +1,143 @@
+
+Specifically, Adobe Express will be setting the following headers:
+
+- `Cross-Origin-Embedder-Policy: credentialless`
+- `Cross-Origin-Opener-Policy: same-origin`
+- `Cross-Origin-Resource-Policy: same-site`
+
+This change is being done to support Adobe Express’ use of certain browser capabilities requiring cross-origin isolation.
+
+## Impact
+
+This change may impact your add-on’s access to external resources, especially if it relies on iframes to display content or support payment flows. It could also impact  `fetch` or image requests from external sources. In these cases, users may see missing content or be presented with silent failures if your add-on can’t load a remote resource. Since this results in a poor experience, developers must ensure that their add-ons work in the above environment.
+
+Please note that this change affects *all* add-ons, even those not published in the add-on marketplace (i.e., private and internally distributed add-ons).
+
+Currently, this change *only impacts* Chromium-based browsers (e.g., Chrome, Edge, Opera, etc.). Firefox and Safari browsers are currently exempt. If you've developed a an add-on with mobile support, this would apply to add-ons running on Android devices as well.
+
+## Types of failures
+
+Some failures will be more evident than others, but all will negatively impact the user experience of your add-on.
+
+- If a nested iframe fails to load, Chrome displays a very obvious error message inside the iframe indicating that the domain "refused to connect".
+- If an image fails to load, you may notice missing images in your add-on’s user interface. You should also see failures in the Network section of the browser’s developer tools.
+- If a network call fails due to JavaScript code, you should see warning and error messages in the browser’s developer tools.
+
+## Testing your add-on
+
+Until Adobe Express enables these headers by default, you must configure your local development environment to simulate these changes.
+
+### Apply local header overrides
+
+The developer tools in Chromium-based browsers on the desktop allow you to specify header overrides. Using this, you can simulate the headers that Adobe Express will enable to test whether your add-on has any problems in this environment.
+
+<InlineAlert slots="header, text" variant="info"/>
+
+Mobile add-ons
+
+You cannot test this on mobile devices. You should test your add-on on a desktop web browser. Any issues you run into would also appear on mobile devices, and any fixes you apply would also apply to mobile users.
+
+To enable this environment yourself, perform the following steps:
+
+First, launch your browser’s developer tools and navigate to the "Network" tab, and then navigate to [Adobe Express](https://new.express.adobe.com) in the browser. Your network panel should fill up with a lot of network traffic, like this:
```

Comment:
Can we have this as step by step? 

1. Open Developer Tools:
Launch your Chromium-based browser (e.g., Chrome, Edge, Opera).
Open the developer tools by pressing F12 or Ctrl+Shift+I (Windows/Linux) or Cmd+Option+I (Mac).

2. Navigate to the Network Tab:
In the developer tools, click on the **Network** tab.

And so on...